tests/test.configurations.js

```diff
@@ -27,10 +27,32 @@ global.harnessConfigs = {
     initialDaemonListenerPort: 50000,
     initialDaemonHttpPort: 8080,
 
-    daemonStartTimeout: 3000,
-    keyCreationTimeoutMultiplier: process.env.KEY_CREATION_TIMEOUT_MULTIPLIER ? process.env.KEY_CREATION_TIMEOUT_MULTIPLIER : 750, // time allotted per key created in before hooks of dynamically generated tests
-    defaultTestTimeout: process.env.TEST_TIMEOUT ? process.env.TEST_TIMEOUT : 5000,
-    defaultBeforeHookTimeout: process.env.BEFORE_HOOK_TIMEOUT ? process.env.BEFORE_HOOK_TIMEOUT : 25000,
-    defaultAfterHookTimeout: process.env.AFTER_HOOK_TIMEOUT ? process.env.AFTER_HOOK_TIMEOUT : 10000,
-    clientOperationTimeout: 10000
+    keyCreationTimeoutMultiplier: process.env.KEY_CREATION_TIMEOUT_MULTIPLIER ? process.env.KEY_CREATION_TIMEOUT_MULTIPLIER : 5000, // time allotted per key created in before hooks of dynamically generated tests
+    daemonStartTimeout: 5000,
 };
+
+Object.defineProperties(global.harnessConfigs, {
+    viewChangeTimeout: {
+        value: multiplyKeyCreationTimeoutBy.call(harnessConfigs, 300)
+    },
+    clientOperationTimeout: {
+        value: multiplyKeyCreationTimeoutBy.call(harnessConfigs, 100)
+    },
+    defaultTestTimeout: {
+        value: multiplyKeyCreationTimeoutBy.call(harnessConfigs, 50)
+    },
+    defaultBeforeHookTimeout: {
+        value: multiplyKeyCreationTimeoutBy.call(harnessConfigs, 250)
```

Comment:
These all seem a bit high, were they empirically determined?

Reply:
Yeah, I arrived at those numbers by basing it on my local. What the total timeout that I needed for the tests to pass divided by the average operation time.